src/components/articles/ArticlesList.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Article from './Article';
+
+const ArticlesList = ({ articles }) => (
+    <ul className="articles-grid">
+        {articles.map((article, index) => (
+            <li className={index === 0 ? "one_third" : "one_third secondary-article"} key={index}>
```

Comment:
las clase "one_third" no se usa en ninguna parte, entiendo cual es el objetivo de esta clase pero aun no esta implementada en css, mejor ponerlo así 
```suggestion
            <li className={index  > 0 && "secondary-article"} key={index}>
```